.circleci/base-generated-config.yaml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+jobs:
+  test-example:
+    parameters:
+      example-app-name:
+        type: string
+      path:
+        type: string
+    machine:
+      image: ubuntu-2004:current
+      docker_layer_caching: true
+    environment:
+      BUILDING_BLOCK: digital-registries
+      TESTED_APPLICATION: << parameters.example-app-name >> 
+      TEST_RESULT_REPORT_FILENAME: example_result.message
+    steps:
+      - checkout
+      - run:
+          name: Execute tests
```

Comment:
Would be cool to include the name of the application we are testing the step names - leverage the fact that we are generating this config.